CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### v0.6.x
 - add: Tutorial for saving live timeseries data (e.g., forecast) in context broker and timeseries database ([#336](https://github.com/RWTH-EBC/FiLiP/pull/363))
 - add: Tutorial for using session object ([#370](https://github.com/RWTH-EBC/FiLiP/pull/370))
+- add: New features to validate and update relationships for V2 entities ([#379](https://github.com/RWTH-EBC/FiLiP/pull/380))
```

Comment:
Should this be moved to 0.6.2?

Reply:
Yes, I will move it to the next release.